src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,50 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static addAnimal(animal) {
+    if (this.alive === undefined) {
+      this.alive = [];
+    }
+    this.alive.push(animal);
+  }
+
+  static removeAnimal(animal) {
+    this.alive = this.alive.filter(liveAnimal => animal !== liveAnimal);
+  }
```

Comment:
```suggestion
 Animal.alive.push(this);
```

that can all be replaced by a single line